practice-one/src/assets/css/global.css

```diff
@@ -9,6 +9,9 @@
   --gray-color: #7d7d7d;
   --brown-color: #ebcb80;
   --orange-color: #e8b229;
+  --fz-primary: 18px;
+  --fz-secondary: 16px;
+  --fz-thirdary: 12px;
```

Comment:
@HoangLiarPhuongLe any update for naming?

Reply:
Yes, I updated for naming

![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/aa27b7c1-d544-4586-b955-27ef85ad7013)